packages/thunderstore-api/README.md

```diff
@@ -0,0 +1,13 @@
+# thunderstore-api
+
+Thunderstore API client
+
+## Scripts
+
+- `yarn run build`: Builds the project
+- `yarn run dev`: Builds the project & watches files for changes, triggering a rebuild
+- `yarn run jest`: run test suite once
+- `yarn run test`: runs test suite & watches files for changes, triggering rerun
```

Comment:
I feel like conventionally this behavior would be behind a command named `watch` or such. Or at the very least the `test` command itself should be a one-off run you can use e.g. on CI pipelines.

Reply:
Changed the usage across the monorepo in a separate commit.